### DISCUSSION TITLE
Where to Post MathJax/Github Issues

### DISCUSSION THREAD

**Discussion by DLu** [community/community#23108] [Projects and Issues]
<p>Where is the proper place to post issues with how MathJax is rendered on Github? I don’t believe it is a problem with MathJax itself, but how Github is parsing the dollar signs.</p>
<p><a href="https://gist.github.com/DLu/2da201d4135e0d74f552b9b729fa024e" rel="noopener nofollow ugc">My example</a></p>

**Comment by jfullstackdev** [comment]
<p>if we are referring to the same project, then I think this is the repo,</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/mathjax/MathJax">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/mathjax/MathJax" target="_blank" rel="noopener">GitHub</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/47092464/181074885-1a68cad3-9f64-4ed6-b36a-029ae6bb8e44.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/f/7/f712160fa500d6d234153fd7689b103d6c54cbe3_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/f/7/f712160fa500d6d234153fd7689b103d6c54cbe3_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/f/7/f712160fa500d6d234153fd7689b103d6c54cbe3.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/f/7/f712160fa500d6d234153fd7689b103d6c54cbe3_2_10x10.png"></div>

<h3><a href="https://github.com/mathjax/MathJax" target="_blank" rel="noopener">GitHub - mathjax/MathJax: Beautiful and accessible math in all browsers</a></h3>

  <p>Beautiful and accessible math in all browsers. Contribute to mathjax/MathJax development by creating an account on GitHub.</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>raise the question by creating an issue there</p>

**Comment by DLu** [comment]
<p>That is the upstream repository, but they don’t perform the escaping of the Markdown files that Github has, so it doesn’t seem like the right place to open an issue.</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>how about here ?</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/github-community/community/discussions/categories/profile">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/github-community/community/discussions/categories/profile" target="_blank" rel="noopener">GitHub</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/47092464/181074889-97e960d4-504c-4871-a963-0e106686a915.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/2/4/2401d7bd09718eb3e29d860a1ec5a5c99a969161_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/2/4/2401d7bd09718eb3e29d860a1ec5a5c99a969161_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/2/4/2401d7bd09718eb3e29d860a1ec5a5c99a969161.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/2/4/2401d7bd09718eb3e29d860a1ec5a5c99a969161_2_10x10.png"></div>

<h3><a href="https://github.com/github-community/community/discussions/categories/profile" target="_blank" rel="noopener">Discussions · github-community/community</a></h3>

  <p>Public feedback discussions for: GitHub for Mobile, GitHub Discussions, GitHub Codespaces, GitHub Sponsors, GitHub Issues and more! - Discussions · github-community/community</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>it’s feedback so that is the Prod Team to better track if that is a request</p>
